tests/humanize-utils/test_humanize_seconds.py

```diff
@@ -0,0 +1,45 @@
+import pytest
+
+from eth_utils import humanize_seconds
+
+
+SECOND = 1
+MINUTE = 60
+HOUR = 60 * 60
+DAY = 24 * HOUR
+YEAR = 365 * DAY
+MONTH = YEAR // 12
+WEEK = 7 * DAY
+
+
+@pytest.mark.parametrize(
+    "seconds,expected",
+    (
+        (0, "0s"),
+        (1, "1s"),
+        (60, "1m"),
```

Comment:
I guess it's worth adding one just below the boundary too, like `59s`. I don't see any problems, more for preventing regressions.
Also: `59m59s` etc